nshimiyejayd/my-brand
# Feat section 1

#### What does this PR do?
This PR implements My Brand section one

#### Description of Task to be completed?
- Add Home page
- Add About me page
- Add Contact page
- Add Portfolio page
- Add Skills page
- Add Login

#### How should this be manually tested?
- Clone the [repo](https://github.com/nshimiyejayd/my-brand) in your computer
- Checkout branch `feat-section-1`
- Open index.html in your browser to check all pages

#### What are the relevant pivotal tracker stories?
[trello-board](https://trello.com/b/TmnFdrKg/atlp-c-25-my-brand)
